modules/promotion/src/Annotation/PromotionOffer.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Drupal\commerce_promotion\Annotation;
+
+use Drupal\Component\Annotation\Plugin;
+
+/**
+ * Defines a Promotion Offer item annotation object.
+ *
+ * Plugin namespace: Plugin\Commerce\PromotionOffer.
+ *
+ * @Annotation
+ */
+class PromotionOffer extends Plugin {
+
+
+  /**
+   * The plugin ID.
+   *
+   * @var string
+   */
+  public $id;
+
+  /**
+   * The label of the plugin.
+   *
+   * @var \Drupal\Core\Annotation\Translation
+   *
+   * @ingroup plugin_translatable
+   */
+  public $label;
+
+  /**
+   * The name of the module providing the type.
+   *
+   * @var string
+   */
+  public $module;
+
+  /**
+   * An array of context definitions describing the context used by the plugin.
+   *
+   * The array is keyed by context names.
+   *
+   * @var \Drupal\Core\Annotation\ContextDefinition[]
+   */
+  public $context = [];
+
+  /**
+   * The target this action applies to.
+   *
+   * For example, this should be 'order' or 'line_item'.
```

Comment:
Because it made the annotation seem long and ugly. `targetEntityType` over `target`. But we can change it to being more verbose